platform/ui/src/components/Tooltip/Tooltip.jsx

```diff
@@ -39,7 +39,7 @@ const Tooltip = ({ content, isSticky, position, tight, children }) => {
     }
   };
 
-  const isOpen = isSticky || isActive;
+  const isOpen = (isSticky || isActive) && !isDisabled;
```

Comment:
Interesting. I'm guessing we have to do it this way because the `Tooltip` wraps it's children? Works for me, but can we do an inline comment above the prop?

```js
/** prevents tooltip from rendering despite hover/active/sticky */
isDisabled: false,
```